pkgs/development/python-modules/kivy/default.nix

```diff
@@ -0,0 +1,62 @@
+{ lib
+, buildPythonPackage, fetchPypi, fetchpatch
+, pkg-config, cython
+, kivy-garden, requests, docutils, pygments
+, mesa, mtdev, SDL2, SDL2_image, SDL2_ttf, SDL2_mixer, gst_all_1
+, pillow
+}:
+
+buildPythonPackage rec {
+  pname = "Kivy";
+  version = "2.0.0rc4";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1l19kfsprgrh2a19s11zbwgjdgbwh4lvnwg27rri01kgvsisbhqn";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+    cython
+    docutils
+  ];
+
+  buildInputs = [
+    kivy-garden
+
+    mesa
+    mtdev
+    SDL2
+    SDL2_image
+    SDL2_ttf
+    SDL2_mixer
+
+    # NOTE: The degree to which gstreamer actually works is unclear
+    gst_all_1.gstreamer
+    gst_all_1.gst-plugins-base
+    gst_all_1.gst-plugins-good
+    gst_all_1.gst-plugins-bad
+  ];
+
+  KIVY_HOME = "/build";
```

Comment:
I think there's something like `KIVY_NO_CONFIG` which sounds less hacky.

Reply:
please try to avoid referencing the build directory. references to it will cause it to fail

Reply:
Yep, fixed it.